lib/generators.nix

```diff
@@ -177,13 +177,14 @@ rec {
 
   # PLIST handling
   toPlist = {}: v: let
```

Comment:
Guess it's a bit late, but I kind of disagree with the casing plist is an acronym. It should be `toPLIST` and `toPropertyList` IMHO.